web/src/modules/app/initSentry.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import buildConfig from './constants/buildConfig'
+
+const initSentry = async () => {
+  if (!buildConfig().featureFlags.sentry) {
+    console.log('Disabling sentry because it was disabled through the build config.')
+    return
+  }
+
+  try {
+    const Sentry = await import(
+      /* webpackChunkName: "sentry" */
+      '@sentry/react'
+    )
+
+    Sentry.init({
+      dsn: 'https://f07e705b25464bbd8b0dbbc0a6414b11@sentry.tuerantuer.org/2',
+      release: `${__BUILD_CONFIG_NAME__}@${__VERSION__}`
```

Comment:
I wonder if it wouldn't be better to use the `appName` from the build config instead of the name.

Reply:
I also guess that the version won't be correct here. You will have to pass it from the environment variables by CircleCI to the webpack config.

Reply:
The thing here is that the appName does not have to stay constant. appName describes only the name of the app on the launch screens on phones afaik.

Reply:
It is also used for translations. I don't think we will and should ever change this. I see your point though.

Reply:
That is alright though. Or at least it does not harm anything.

If we decide to show "Integreat - Great Integreation" on the launch screens, then it would be okish to include that in the translations.